recipes/gpm-imerg/recipe.py

```diff
@@ -0,0 +1,34 @@
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern
+import pandas as pd
+import aiohttp
+
+#TODO: replace with ENV vars
+username = password = 'pangeo@developmentseed.org'
+
+def make_filename(time):
+    input_url_pattern = (
+        "https://arthurhouhttps.pps.eosdis.nasa.gov/gpmdata/{yyyy}/{mm}/{dd}/imerg/3B-HHR.MS.MRG.3IMERG.{yyyymmdd}-S{sh}{sm}00-E{eh}{em}59.{MMMM}.V06B.HDF5"
+    ).format(
+        yyyy=time.strftime("%Y"),
+        mm = time.strftime("%m"),
+        dd = time.strftime("%d"),
+        yyyymmdd=time.strftime("%Y%m%d"),
+        sh = time.strftime("%H"),
+        sm = time.strftime("%M"),
+        eh = time.strftime("%H"),
+        em = (time+pd.Timedelta("29 min")).strftime("%M"),
+        MMMM = f'{(time.hour*60 + time.minute):04}'
+    )
+    return input_url_pattern
+
+dates = pd.date_range("2000-06-01T00:00:00", "2021-05-31T23:59:59", freq="30min")
+time_concat_dim = ConcatDim("time", dates, nitems_per_file=1)
+pattern = FilePattern(make_filename, time_concat_dim)
+
+recipe = XarrayZarrRecipe(
+    pattern, 
+    xarray_open_kwargs={'group': 'Grid', 'drop_variables': ['time_bnds', 'lon_bnds', 'lat_bnds']},
```

Comment:
One more comment--why drop these variables. They are useful metadata.

A better approach would be
```suggestion
    xarray_open_kwargs={'group': 'Grid', 'decode_coords': 'all'},
```
which would promote them to coordinates.

Reply:
I was following @davidbrochart on that, forget if it caused issues. Happy to adjust, though the lon_bnds and lat_bnds will always be the same for every chunk afaik.